docs/spec.md

```diff
@@ -3652,29 +3652,35 @@ More formally, `result[result_index]` is defined as:
 
 #### Inputs
 
-| Label | Name                | Type                                         | Constraints      |
-|-------|---------------------|----------------------------------------------|------------------|
-| (I1)  | `operand`           | tensor                                       | (C1), (C2), (C4) |
-| (I2)  | `padding_value`     | 0-dimensional tensor                         | (C1)             |
-| (I3)  | `edge_padding_low`  | 1-dimensional tensor constant of type `si64` | (C2), (C4)       |
-| (I4)  | `edge_padding_high` | 1-dimensional tensor constant of type `si64` | (C2), (C4)       |
-| (I5)  | `interior_padding`  | 1-dimensional tensor constant of type `si64` | (C2-C4)          |
+| Label | Name                | Type                                                | Constraints |
+|-------|---------------------|-----------------------------------------------------|-------------|
+| (I1)  | `operand`           | tensor or per-tensor quantized tensor               | (C1), (C3)  |
+| (I2)  | `padding_value`     | 0-dimensional tensor or per-tensor quantized tensor | (C4-C6)     |
+| (I3)  | `edge_padding_low`  | 1-dimensional tensor constant of type `si64`        | (C1), (C3)  |
+| (I4)  | `edge_padding_high` | 1-dimensional tensor constant of type `si64`        | (C1), (C3)  |
+| (I5)  | `interior_padding`  | 1-dimensional tensor constant of type `si64`        | (C1-C3)     |
 
 #### Outputs
 
-| Name     | Type   | Constraints |
-|----------|--------|-------------|
-| `result` | tensor | (C1)        |
+| Name     | Type                                  | Constraints |
+|----------|---------------------------------------|-------------|
+| `result` | tensor or per-tensor quantized tensor | (C3-C6)     |
 
 #### Constraints
 
-* (C1) `element_type(operand) = element_type(padding_value) =
-  element_type(result)`.
-* (C2) `size(edge_padding_low) = size(edge_padding_high) =
+* (C1) `size(edge_padding_low) = size(edge_padding_high) =
   size(interior_padding) = rank(operand)`.
-* (C3) `0 <= interior_padding`.
-* (C4) `shape(result) = shape(operand) + edge_padding_low +
+* (C2) `0 <= interior_padding`.
+* (C3) `shape(result) = shape(operand) + edge_padding_low +
   max(shape(operand) - 1, 0) * interior_padding + edge_padding_high`.
+* (C4) If the operation uses non-quantized tensors:
+  * `element_type(operand) = element_type(padding_value) =
+  element_type(result)`.
+* If the operation uses quantized tensors:
```

Comment:
for pad op, it might be safe to consider this operator required the same quantization parameter.

Reply:
Per
```
* (C1) `element_type(operand) = element_type(padding_value) =
  element_type(result)`.
```

we make sure that the quantization parameters are the same. I note that `element_type` is an overloaded auxiliary function applicable to both `TensorType` and `QuantizedTensorType`. PTAL.